src/applications/disability-benefits/all-claims/utils.jsx

```diff
@@ -267,6 +268,38 @@ export function transformProviderFacilities(providerFacilities) {
   }));
 }
 
+export function concatIncidentLocationString(incidentLocation) {
+  return [
+    incidentLocation.city,
+    incidentLocation.state,
+    incidentLocation.country,
+    incidentLocation.additionalDetails,
+  ]
+    .filter(locationField => locationField && locationField.length > 0)
+    .join(', ');
+}
+
+export function getFlatIncidentKeys() {
+  const incidentKeys = [];
+
+  for (let i = 0; i < PTSD_INCIDENT_ITERATION; i++) {
+    incidentKeys.push(`incident${i}`);
+  }
+  for (let i = 0; i < PTSD_INCIDENT_ITERATION; i++) {
+    incidentKeys.push(`secondaryIncident${i}`);
+  }
+
+  return incidentKeys;
+}
+
+export function getPtsdChangeFieldTitles(changeFields, formConfig) {
+  return Object.keys(changeFields)
+    .filter(
+      key => key !== 'other' && key !== 'otherExplanation' && formConfig[key],
+    )
+    .map(key => formConfig[key]['ui:title']);
```

Comment:
I feel like a react component or snippet wouldn't work as intended for the 781 form so I added a check to only support string `ui:title` values.

Reply:
I'm also uncomfortable with relying on the `uiSchema`. What if we made a map that can be used both in the `uiSchema` and in this function?

Do we have to send the api the exact title of the boxes they've selected? This might be something best left to the PDF generator or the api if it affects what gets put on the PDF. Separation of concerns and all that, y'know?